README.md

```diff
@@ -8,52 +8,65 @@ A loading indicator component for EmberJS that animates across the top of the vi
 
 ## Installation
 
-`npm install ember-cli-loading-slider --save-dev`
+`ember install ember-cli-loading-slider`
 
 ## Usage
 
 Add the component to your application template:
 
-    {{loading-slider isLoading=loading duration=250}}
-
-Alternatively you may try the more complex mode. In this mode an array of colors
-is required `hexColorsArray: ['#000', '#fff']`:
-
-    {{loading-slider isLoading=loading expanding=true color=hexColorsArray}}
+    {{loading-slider duration=250}}
```

Comment:
@jerel I was just sitting here and trying to write a proper documentation here but I can no come up with a meaningful version. I'll give it a second try tonight.